packages/client/helpers/generator-test/templates/tests.ts.tpl

```diff
@@ -1,7 +1,8 @@
 import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from '@prisma/client'
 
-// @ts-ignore this is just for type checks
-declare let prisma: import('@prisma/client').PrismaClient
+declare let prisma: PrismaClient
```

Comment:
Is this change now required or is it just a code style change to reduce the number of `// @ts-ignore` directives?

Reply:
It is necessary if we want to use `NewPrismaClient ` type. For some reason I could not get to accept neither `import('@prisma/client').PrismaClient` nor `typeof import('@prisma/client').PrismaClient`